### ISSUE TITLE
The requested URL not found

### ISSUE THREAD

**Issue by raphael-evolutap** [Xammie/mailbook#36]
Hello, I'm getting this error: 

Not Found
The requested URL /mailbook/content/App\Mail\MailbookMail was not found on this server.

Laravel Version: 9.47.0
PHP Version: 8.1.0 
Windows 10

**Comment by raphael-evolutap** [comment]
I'm using laragon to serve, but it worked with **artisan serve**

Thanks

**Comment by Xammie** [comment]
I'm not able to test on windows right now. Are you able to verify that is the actual URL that is requested? The correct URL should look something like this

```
/mailbook/content/App%5CMail%5CMailbookMail
```

**Comment by rcerljenko** [comment]
same happens with me... works with artisan serve but doesn't work with XAMPP

**Comment by MarienMupenda** [comment]
<img width="960" alt="image" src="https://user-images.githubusercontent.com/57844897/212125220-f69f0600-1076-45e0-b493-676ff3a64c3e.png">

Facing the same issue.

Laravel Version: 9.45.0
PHP Version: 8.0
Windows 11
Laragon 5.0.0

**Comment by Xammie** [comment]
I've fixed this issue in #37. Before I release it I would appreciate if someone could verify if it works by installing the following branch.

```
composer require --dev xammie/mailbook dev-fix-laragon-404
```

**Comment by raphael-evolutap** [comment]
Tested here and it worked. Thankx man!

Awesome package

**Comment by MarienMupenda** [comment]
> I've fixed this issue in #37. Before I release it I would appreciate if someone could verify if it works by installing the following branch.
> 
> ```
> composer require --dev xammie/mailbook dev-fix-laragon-404
> ```

<img width="960" alt="image" src="https://user-images.githubusercontent.com/57844897/212262189-5482d55a-2341-4173-984b-c691ab3c78bf.png">

Tested here and it worked. Thanks!

Wery awesome package.

**Comment by Xammie** [comment]
Thanks for testing, glad you like it. I've released this fix in version 1.5.2
